devtools/test/src/org/labkey/test/params/devtools/SecondaryAuthenticationTOTPProvider.java

```diff
@@ -0,0 +1,33 @@
+package org.labkey.test.params.devtools;
```

Comment:
You should be able to put this class in the `mfa` repository.
```suggestion
package org.labkey.test.params.mfa;
```

Reply:
I added this class here, because everything was part of devtools

Still makes sense to move?

Reply:
Yes, it should move. `SecondaryAuthenticationProvider` is in here because the only secondary auth test we had previously was for the trivial one defined in the `devtools` module